Dockerfile

```diff
@@ -1,4 +1,27 @@
-FROM alpine:3.18.4
+FROM alpine:3.18.4 as alpine
+
+RUN apk add --no-cache \
+    ca-certificates \
+    tzdata
+
+# initial default temp dir
+RUN mkdir /tmp/reaper
+
+# minimal image
+FROM scratch
+COPY --from=alpine \
+    /etc/ssl/certs/ca-certificates.crt \
+    /etc/ssl/certs/ca-certificates.crt
+COPY --from=alpine \
+    /usr/share/zoneinfo \
+    /usr/share/zoneinfo
+COPY --from=alpine \
+    /tmp/reaper \
+    /tmp/reaper
+
 COPY reaper /usr/local/bin/reaper
 
-CMD ["reaper"]
\ No newline at end of file
+WORKDIR /reaper
+
+ENTRYPOINT ["reaper","-c","/reaper/config.yaml"]
```

Comment:
Should remove the "-c" parameter as it is already default. See [here](https://github.com/LeslieLeung/reaper/blob/0214f53463a18fd8bad8580bb4d881d1b4371985/cmd/root.go#L29C1-L29C1).

Reply:
current defautlt config path used in docker is `/reaper/config.yaml`, which is different with the default set in program. 
maybe set the same defautlt config path in docker container?

Reply:
> current defautlt config path used in docker is `/reaper/config.yaml`, which is different with the default set in program. maybe set the same default config path in docker container?

Updated.